go/libkb/errors.go

```diff
@@ -2704,6 +2704,25 @@ func (c ChainLinkBadUnstubError) Error() string {
 
 //============================================================================
 
+// AppOutdatedError indicates that an operation failed because the client does
+// not support some necessary feature and needs to be updated.
+type AppOutdatedError struct {
+	cause error
+}
+
+func NewAppOutdatedError(cause error) AppOutdatedError {
+	return AppOutdatedError{cause: cause}
+}
+
+func (e AppOutdatedError) Error() string {
+	if e.cause != nil {
+		return fmt.Sprintf("AppOutdatedError: %v", e.cause.Error())
```

Comment:
This might need to be user-friendlier, if it's being displayed on the CLI or promoted to the UI. Also, you might need to make this error exportable if the GUI can digest it. For an example of this, see `ChatConvExistsError`. Might need to make a change to `keybase/keybase` to reserve a status code if there isn't one that already exists

Reply:
I am ticketing this. At the moment it is not displayed anywhere as the proof verifier is not plumbed.